### PR TITLE
Adding christmas planner page & Hero slot

### PR DESCRIPTION
**Pull request by johnplayer1982**
Adding Christmas planner page and Hero slot on homepage

### CODE REVIEW

File: app/views/christmas_money_planner/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<% content_for(:alternate_link) do %>
+  <%= link_to(alternate_url, lang: alternate_locale, class: "t-#{alternate_locale}-link") do %>
+    <span class="icon icon--globe"></span>
+    <%= t("locales.#{alternate_locale}") %>
+  <% end %>
+<% end %>
+
+<div class="l-constrained">
+  <div class="l-container-tool">
+    <!--<h1><%= t('christmas_money_planner.page_heading') %></h1>
```
Comment (benbarnett) [inline app/views/christmas_money_planner/show.html.erb]: Probably should remove these if they're not needed

**Comment by benbarnett** [comment]
@johnplayer1982 that image can be compressed a little more I think, have you tried [ImageOptim](https://imageoptim.com/) before?

File: app/views/christmas_money_planner/show.html.erb (hunk 1 above)
Comment (johnplayer1982) [inline app/views/christmas_money_planner/show.html.erb]: Good catch, they have now been removed and their corresponding entries in en & cy .yml

**Comment by johnplayer1982** [comment]
Thanks for the link to ImageOptim! Great little tool, saved 10%, image updated on PR

**Comment by benbarnett** [comment]
Nice one, looks good! :+1:

**Comment by pmcostello** [comment]
:+1: looks good :shipit:

**Pull request merged**